### PR TITLE
explisit set style and explode

### PR DESCRIPTION
**Pull request by sondrev**

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -865,14 +922,17 @@ components:
...
+                format: uri
               name:
                 type: string
         description:
           type: string
         doi:
           type: string
-        id:
+          format: uri
+        identifier:
```
Comment (sondrev) [inline docs/openapi.yaml:933]: I could not find a "id" on publication. It should be "identifier" - right?

Comment (ketilaa) [reply docs/openapi.yaml:933]: PublicationResponse implements WithId that adds the `id` field.

File: docs/openapi.yaml [hunk 2]
```diff
@@ -304,7 +314,11 @@ paths:
         name: publicationIdentifier
         schema:
           type: string
+          format: uuid
```
Comment (ketilaa) [inline docs/openapi.yaml:317]: Identifier is not full URI, just the SortableIdentifier (UUID-like)

File: docs/openapi.yaml [hunk 3]
```diff
@@ -341,12 +355,20 @@ paths:
...
```
Comment (ketilaa) [inline docs/openapi.yaml:358]: probably not correct

File: docs/openapi.yaml [hunk 4]
```diff
@@ -341,12 +355,20 @@ paths:
...
         required: true
+        style: simple
+        explode: false
+        description: UUID of the publication
       - in: path
         name: ticketIdentifier
         schema:
           type: string
+          format: uuid
```
Comment (ketilaa) [inline docs/openapi.yaml:367]: probably not correct

File: docs/openapi.yaml [hunk 5]
```diff
@@ -463,12 +485,20 @@ paths:
         name: publicationIdentifier
         schema:
           type: string
+          format: uuid
```
Comment (ketilaa) [inline docs/openapi.yaml:488]: probably not correct

File: docs/openapi.yaml [hunk 6]
```diff
@@ -463,12 +485,20 @@ paths:
...
         required: true
+        style: simple
+        explode: false
+        description: UUID of the publication
       - in: path
         name: ticketIdentifier
         schema:
           type: string
+          format: uuid
```
Comment (ketilaa) [inline docs/openapi.yaml:497]: probably not correct

File: docs/openapi.yaml [hunk 7]
```diff
@@ -505,17 +536,29 @@ paths:
         name: publicationIdentifier
         schema:
           type: string
+          format: uuid
```
Comment (ketilaa) [inline docs/openapi.yaml:539]: probably not correct

File: docs/openapi.yaml [hunk 8]
```diff
@@ -505,17 +536,29 @@ paths:
...
         required: true
+        style: simple
+        explode: false
+        description: UUID of the publication
       - in: path
         name: ticketIdentifier
         schema:
           type: string
+          format: uuid
```
Comment (ketilaa) [inline docs/openapi.yaml:548]: probably not correct

File: docs/openapi.yaml [hunk 9]
```diff
@@ -505,17 +536,29 @@ paths:
...
         required: true
+        style: simple
+        explode: false
+        description: UUID of the ticket
       - in: path
         name: messageIdentifier
         schema:
           type: string
+          format: uuid
```
Comment (ketilaa) [inline docs/openapi.yaml:557]: probably not correct

File: docs/openapi.yaml [hunk 10]
```diff
@@ -580,7 +623,11 @@ paths:
         name: publicationIdentifier
         schema:
           type: string
+          format: uuid
```
Comment (ketilaa) [inline docs/openapi.yaml:626]: I know we do some "magic" with our identifiers to make them sortable, so it is not a standard UUID.

File: docs/openapi.yaml [hunk 11]
```diff
@@ -865,14 +922,17 @@ components:
...
                 type: string
         description:
           type: string
         doi:
           type: string
-        id:
+          format: uri
+        identifier:
           type: string
+          format: uri
```
Comment (ketilaa) [inline docs/openapi.yaml:935]: identifier is typically not an uri

File: docs/openapi.yaml [hunk 12]
```diff
@@ -1049,13 +1113,15 @@ components:
...
               type: string
+              format: uri
             name:
               type: string
         reference:
           type: object
           properties:
             doi:
               type: string
+              format: uuid
```
Comment (ketilaa) [inline docs/openapi.yaml:1124]: doi is a `uri`

**Review by ketilaa — COMMENTED** [review]
Generally, `id`, `xxxId`, `doi` and `handle` is `uri`.
Identifier is just string. I think we extend UUID with some time info to make it sortable, so better not set a format there.

**Review by ketilaa — APPROVED** [review]

**Pull request merged**